### DISCUSSION TITLE
GitHub REST API - get current user's enterprise memberships

### DISCUSSION THREAD

**Discussion by mikeclayton** [community/community#29918] [Enterprise] [answered]
I've just started using the GitHub REST API, and I'm trying to find a way to retrieve the list of (Cloud) Enterprises the current / authenticated user is a member of - basically the list that appears when you navigate to https://github.com/settings/enterprises in a browser, but I'm drawing a complete blank with the documentation at https://docs.github.com/en/rest.

(NB: I originally posted this question at https://stackoverflow.com/questions/73319031/github-rest-api-get-current-users-enterprise-memberships but didn't get a response, so reposting here. in the hope someone might have an answer / workaround...)

This is the information I'm trying to retrieve via the API:

![image](https://user-images.githubusercontent.com/1193763/185362905-eceeaceb-9fd8-4ea5-9a14-6a8d05f17a8c.png)

For clarity, I'm authenticating against the API using a Personal Access Token, and I'm trying to retrieve the list of Enterprises for the user the PAT is associated with.

Is there a specific API endpoint that returns this information? Or is there a way to derive it from another endpoint (e.g. getting the list of organisations for the authenticated user and extracting the enterprise details from them somehow)?

I'm currently using the latest OctoKit.net in C# for some other integration with the GitHub API, in case there's a built-in way to do it, but I'll happily send a bare-bones HTTP request to a uri if one exists.

Any info or pointers greatly appreciated...

Thanks,

Mike

**Comment by byrneh** [comment, marked as answer]
@mikeclayton I looked but could not find a REST/GRAPHQL API to return the authenticated users enterprise memberships either

**Comment by mikeclayton** [comment]
@byrneh - Thanks for looklng - I think I've come to the same conclusion. At least I'm not just missing something obvious :-)...
